### DISCUSSION TITLE
Config Naming Structure

### DISCUSSION THREAD

**Discussion by Spartwo** [likeproblem/RestockRecolour#5] [Ideas]
Texture sets should contain configs named to mirror their stock recolour counterparts for automatic craft compatibility

**Comment by likeproblem** [comment]
Like names inside TURD? So we are "replacing" TURD's recolours with our own

**Reply by Spartwo** [reply]
Say that stock has +Texturset[Terrier_Paint] and this mod has Terrier_Paint

When you uninstall or install restock the existing painted parts will keep their paint setups

But if it's Terrier_Restock_Paint then any changes are going to have people need to redo their craft. As restock and stock paint are mutually exclusive there are no real downsides to this change so early on in development.

**Reply by UncleMateo** [reply]
I didn't really consider this, there's a few parts that will need fixing.  But Spartwo is right, better not to break compatibility, if possible.
I will try and do that structure henceforth, as long as there's not any issues.

**Reply by Spartwo** [reply]
No I was wrong as unfortunately +KSP_TEXTURE_SET[Restock_Paint] also breaks that flow

Do some bulk text replace and yea you could setup with compatibility fairly straight
